src/components/protectedRoute/protectedRoute.tsx

```diff
@@ -0,0 +1,20 @@
+import { useNavigate } from "react-router-dom";
+import { UserAuth } from "../../services/AuthContext/AuthContext";
+
+interface AuthContextProps {
+  children?: any;
```

Comment:
```suggestion
  children: JSX.Element;
```
Давай так пока оформим.